mmv1/products/compute/UrlMap.yaml

```diff
@@ -303,6 +303,7 @@ properties:
           #   - path_matchers.0.default_service
           #   - path_matchers.0.default_url_redirect
           #   - path_matchers.0.default_route_action.0.weighted_backend_services
+          required: true
```

Comment:
Instead of making this field `required`, we can add a note to the `description` of this field that one of `default_service` and `default_url_redirect` should be provided.